src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -39,6 +39,10 @@
     @Column(name = "detail_text")
     private String detailText;
 
+    @Size(max = 5000)
+    @Column(name = "automatic_detail_text")
+    private String automaticDetailText;
```

Comment:
Can you check if there are any performance or memory issues with this new attribute?
We have 21 million feedback entries in the production database, many of them won't ever use this field, so adding this might be problematic and we might want to think about alternative approaches here!

Reply:
I made sure that this column is null by default and only initialized in the cases where there is an original automatic feedback that got changed. I've just looked up the memory and appearently 8 Null take 1byte. Therefore this whole database would grow by 0,33 MB. 

The only other way I can think of right now to store the feedback consistently would be adding another uni-directional table where I don't need to store the Null values. This would add more complexity due to another Table and another Rest Call. Thats why I chose to use another column.

Reply:
what do you mean with "8 Null"? 
what about the performance?